src/components/SiteMapFooter.tsx

```diff
@@ -0,0 +1,87 @@
+import react from "react";
+import SiteMapFooterTextContent from "./SiteMapFooterTextContent";
+import SocialMediaFooterIcons from "./SocialMediaFooterIcons";
+import FooterGotQuestionsBanner from "@/components/FooterGotQuestionsBanner";
+
+export default function SiteMapFooter() {
+  return (
+    <div className="bg-background-site-footer-colour">
+      <div className="px-14 pb-5 gap-2 pt-6 font-sans w-full h-[430px] flex flex-row justify-center text-gray-400 font-[400]">
+        {footerContent.map((content, index) => (
+          <ul className="flex-column text-xs w-56 h-6" key={index}>
+            {content.title}
+            <SiteMapFooterTextContent content={content.body} />
```

Comment:
I would rename this a little, it comes off as text but maybe it's bodyContent

Reply:
addressed.